global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.100",
+    "version": "9.0.100",
```

Comment:
So far we have only supported the LTS versions of .NET and skipped the STS versions. 
I don't know if switching to building with .NET 9 should be included.

Reply:
yes, this was for testing only (fresh formatted pc, .NET 9 only available). You can skip this change.